demo/html/FirstDivision/index.html

```diff
@@ -2,11 +2,11 @@
 <!doctype html>
 <html lang="en"><head>
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<title>Sysl Catalog </title>
+<title>Sysl Catalog</title>
 </head>
 <body>
 <div id='content'>
-<p><a href="../index.html">Back</a></p>
+<p><a href="../index.html">Back</a>  | <a href="">Chat with us</a> | <a href="">New bug or feature request</a></p>
```

Comment:
Hmm, these links should be populated by default after https://github.com/anz-bank/sysl-catalog/pull/73 has been merged in. Can you please rebase this branch and `make install` to make sure you have these changes?